### PR TITLE
add quickstart guide 

### PR DESCRIPTION
**Pull request by sojinkim-icon**

### CODE REVIEW

File: docs/quickstart_p2.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+
+
+## Transaction fees
+
+[Transaction fees explained](step.md).  
+
+- `stepLimit` in transaction request message
+
+  Every transaction request message has a `stepLimit` field. This value can not exceed the "maximun step limit" defined in the Governance SCORE.
```
Comment (sink772) [inline docs/quickstart_p2.md]: maximun -> maximum

File: docs/quickstart_p2.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    "stepLimit": "0x5000000",
+    ...
+}
+```
+
+
+
+## Deploy HelloWorld to the testnet (T-Bears CLI)
+
+Unlike T-Bears emulated environment, on testnet, valid sinature is required to send a transaction. So, every transaction request will require a keystore file to sign it.
```
Comment (sink772) [inline docs/quickstart_p2.md]: sinature -> signature

**Comment by sojinkim-icon** [comment]
thanks. typo correction pushed.

**Comment by sojinkim-icon** [comment]
this closes #33, part 3 will be added when aws quickstart is ready

**Pull request merged**